web/scripts/embed.dart

```diff
@@ -4,8 +4,8 @@
 
 // Currently not in use
 
-import 'package:dart_pad/embed/embed.dart' as embed;
-import 'package:dart_pad/polymer/polymer.dart';
+import '../../lib/embed/embed.dart' as embed;
+import '../../lib/polymer/polymer.dart';
```

Comment:
Looking at this, we may be able to keep embed.dart in the same folder (`web/`) and just move its compiled output. Let's talk in person.